vendorconsent/tcf2/metadata.go

```diff
@@ -61,6 +66,7 @@ const (
 	decisPerOne  = 10
 )
 
+// Created returns the created date stored in bits 6 to 41
```

Comment:
Bits 7 to 42. To be fair, the comment in the method is a bit deceiving due to starting at index 0.

Reply:
ok